reactos/dll/win32/browseui/travellog.cpp

```diff
@@ -172,6 +172,7 @@ HRESULT STDMETHODCALLTYPE CTravelEntry::Update(IUnknown *punk, BOOL fIsLocalAnch
     GetToolTipText(punk, wch);
     TRACE("Updating entry with display name: %S\n", wch);
 
+    ZeroMemory(&windowData, sizeof(WINDOWDATA));
```

Comment:
It seems that ieframe's ITravelLogClient implementations checks some null pointers in WINDOWDATA before trying to dereference them, leading to explorer crash when trying to browse webpage if this structure isn't properly zeroed.